packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -333,9 +351,72 @@ function MapAnalysisImpl(props: ImplProps) {
     selectedOverlayVariable: activeMarkerConfiguration?.selectedVariable,
     overlayConfig: activeOverlayConfig.value,
     outputEntityId: outputEntity?.id,
-    //TO DO: maybe dependentAxisLogScale
+    dependentAxisLogScale:
+      activeMarkerConfiguration &&
+      'dependentAxisLogScale' in activeMarkerConfiguration
+        ? activeMarkerConfiguration.dependentAxisLogScale
+        : false,
+  });
+
+  const { markersData: previewMarkerData } = useStandaloneMapMarkers({
+    boundsZoomLevel: appState.boundsZoomLevel,
+    geoConfig: geoConfig,
+    studyId,
+    filters,
+    markerType,
+    selectedOverlayVariable: activeMarkerConfiguration?.selectedVariable,
+    overlayConfig: activeOverlayConfig.value,
+    outputEntityId: outputEntity?.id,
+    isMarkerPreview: true,
   });
 
+  const continuousMarkerPreview = useMemo(() => {
+    if (!previewMarkerData || !previewMarkerData.length) return;
+    const initialDataObject = previewMarkerData[0].data.map((data) => ({
+      label: data.label,
+      value: 0,
+      ...(data.color ? { color: data.color } : {}),
+    }));
+    const typedData =
+      markerType === 'pie'
+        ? ([...previewMarkerData] as DonutMarkerProps[])
+        : ([...previewMarkerData] as ChartMarkerProps[]);
+    const finalData = typedData.reduce(
+      (prevData, currData) =>
+        currData.data.map((data, index) => ({
+          label: data.label,
+          value: data.value + prevData[index].value,
+          ...('color' in prevData[index]
+            ? { color: prevData[index].color }
+            : 'color' in data
+            ? { color: data.color }
+            : {}),
+        })),
+      initialDataObject
+    );
+    if (markerType === 'pie') {
+      return (
+        <DonutMarkerStandalone
```

Comment:
Ideally we need to add something like `markerLabel: kFormatter(finalData.reduce(...sum the values...))` to the props here, so that the number formatting is the same (without `markerLabel`, the marker components fall back to doing a sum reduce of the values and render the number as a plain old number).

But this isn't very nice. It would be better I think to get rid of the `markerLabel` prop from the marker components pass a formatting function to the markers instead.  (I added `markerLabel` - my bad!)  But we could do this as technical debt if you prefer.

Aiming for this (hacked in dev tools)
![image](https://github.com/VEuPathDB/web-monorepo/assets/308639/0d2d1411-b032-4ea4-a513-5e55208a4ce5)